### PR TITLE
adding get_timer_emitter method and adding dimensions to creation of counters

### PR DESCRIPTION
**Pull request by neoeahit**
…ounters and timers

### CODE REVIEW

File: kafka/metrics/dict_reporter.py [hunk 1]
```diff
@@ -85,5 +85,8 @@ def close(self):
     def record(self, emitter, value, timestamp):
         pass
 
-    def get_emitter(self, metric):
+    def get_counter_emitter(self, metric, default_dimensions = None):
```
Comment (ny2ko) [inline kafka/metrics/dict_reporter.py]: Remove spaces ie default_dimensions=None

File: kafka/metrics/dict_reporter.py [hunk 2]
```diff
@@ -85,5 +85,8 @@ def close(self):
...
+        pass
+
+    def get_timer_emitter(self, metric, default_dimensions = None):
```
Comment (ny2ko) [inline kafka/metrics/dict_reporter.py]: Ditto on the spaces

**Comment by ecanzonieri** [comment]
Can you add more details about this pr? I don't see get_timer_emitter call in the library? Does that mean all metrics are always emitted by counter?

**Comment by neoeahit** [comment]
Closing this pull request, in favor of the other one.

**Pull request closed without merging**